test/shell/test_scala_config.sh

```diff
@@ -18,7 +18,7 @@ test_classpath_contains_2_13() {
 
 test_scala_config_content() {
   bazel build --repo_env=SCALA_VERSION=0.0.0 @io_bazel_rules_scala_config//:all 2> /dev/null
-  grep "SCALA_MAJOR_VERSION='0.0'" $(bazel info output_base)/external/io_bazel_rules_scala_config/config.bzl
+  grep "SCALA_MAJOR_VERSION='0.0'" $(bazel info output_base)/external/*/config.bzl
```

Comment:
when built with bzlmod target will be in sth like:
`/external/~main_non_module_deps~io_bazel_rules_scala_config/config.bzl`